web/profiles/joinup/joinup.post_update.php

```diff
@@ -178,3 +178,16 @@ function joinup_post_update_legal() {
     ],
   ])->save();
 }
+
+/**
+ * Sets the default weight for specific og actions with configuration.
+ */
+function joinup_post_update_set_default_og_action_weights() {
+  $config_factory = \Drupal::configFactory();
+  $action = $config_factory->getEditable('system.action.og_membership_remove_single_role_action.facilitator');
+  $data = $action->getRawData();
+  $data['weight'] = 1;
+
+  $action->setData($data);
+  $action->save();
```

Comment:
This is because of the `config_sync` module. The configuration files I provided in the patch were not exported up to now an most probably `config_sync` considers them as part of the configuration already since they already are. I could have reset the snapshot but this is very dangerous as it would override all other config updates from the patches included in the release.
I thought I should not waste much time here since we are planning to ditch the thing.